src/signin/Signin.js

```diff
@@ -77,6 +34,11 @@ class SignIn extends Component {
     }
   }
 
+  toSignup() {
```

Comment:
I don't really like the asymmetric logic for this and the signup component. 

I think it should either always initialize the store subtree on `componentWillMount` (my preference) or on these helper functions (e.g. `toSignup()`).

If you reset the store on `componentWillMount` then this function isn't needed. and line 64 is just:

```
<Text style={styles.signup} onPress={Actions.signup}>Sign Up</Text>
```